.github/workflows/_05_force_version_bump.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d
         with:
           workflow: release-perseverance.yml
-          name: chainflip-backend-bin-ubuntu-22.04
```

Comment:
this was just left over from the old naming when we supported both ubuntus. Now that the last release is using the new version, this can be changed here (and now the version bump check works 🚀 )